daffodil-io/src/main/scala/org/apache/daffodil/io/DirectOrBufferedDataOutputStream.scala

```diff
@@ -411,8 +411,24 @@ class DirectOrBufferedDataOutputStream private[io] (
     Assert.invariant(isDirect)
   }
 
+  /**
+   * We need to keep track of what FormatInfo was used as the last write to
+   * this DOS. This way, when we this DOS becomes direct and we start
```

Comment:
Drop extraneous "we", should be "when this DOS".

Reply:
Fixed. Thanks!